keyboards/nopunin10did/jabberwocky/readme.md

```diff
@@ -0,0 +1,21 @@
+# Jabberwocky
+
+![Jabberwocky](https://diplomacyvariants.files.wordpress.com/2021/03/dexterous_jabberwocky.jpg)
+
+A full-size, columnar-staggered, Alice-style keyboard. The Jabberwocky comes in two varieties currently: 
+* Dexterous (with numpad on the right side, pictured above)
+* Sinister (with numpad on the left side)
+
+Both the Dexterous and Sinister use the same firmware.  The numpad and the four keys above it are simply transposed from the right to the left side of the board.
+
+For more information, join the [NoPunIn10Key Discord](https://discord.gg/sku2Y6w).
+
+* Keyboard Maintainer: [Alex Ronke](diplomacyvariants.wordpress.com), also known as NoPunIn10Did. 
+* Hardware Supported: Dexterous Jabberwocky and Sinister Jabberwocky (2021 March)
+* Hardware Availability: Currently by request only
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to jump to the bootloader, as well?

Reply:
Will do. It's a simple reset switch.

Reply:
Just added.